lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py

```diff
@@ -36,9 +36,11 @@
             - Control whether the assigned Public IP remains permanently assigned to the object. If not
               set to 'Static', the IP address my changed anytime an associated virtual machine is power cycled.
         choices:
-            - Dynamic
+            - dynamic
+            - static
             - Static
-        default: Dynamic
+            - Dynamic
```

Comment:
add the Title format and keeping lower case for backward compatibility? can we remove the  Title format in main release?

Reply:
the lower case ones are new added, and keep title format for compatibility. Seems I swap the order or Dynamic and Static and lead to the green highlight change.